code/atom.dm

```diff
@@ -642,7 +642,8 @@
 		if (!hasPDA)
 			. += "<br>*No PDA detected!*"
 
-/atom/proc/MouseDrop_T()
+/atom/proc/MouseDrop_T(dropped, user)
+	SEND_SIGNAL(src, COMSIG_ATOM_MOUSEDROP_T, dropped, user)
```

Comment:
Good idea in theory but in practice people will be confused why their signals is not getting called (it's because something override MouseDrop_T and doesn't call parent). Therefore it might be necessary to do the same thing as was done to attackby and some other procs. Split it into an overridable private part and unoverridable public part. Then put the signal into the public part and make sure nothing except the public part calls the private part.